docs/Filters.md

```diff
@@ -42,69 +42,69 @@ When a filter is evaluated, variables are replaced with the values they represen
 
 Binary operators act on two values:
 
-- `1 + 2`
-- `author.subbed && flags.highlighted`
-- `"long sentence" contains "ten"`
+-   `1 + 2`
+-   `author.subbed && flags.highlighted`
+-   `"long sentence" contains "ten"`
 
 Unary operators act on one value:
 
-- `!author.subbed`
+-   `!author.subbed`
 
 The following operators are available:
 
-| Operator     | Description                                             |
-| ------------ | ------------------------------------------------------- |
-| `&&`         | And                                                     |
-| `||`         | Or                                                      |
-| `!`          | Not                                                     |
-| `==`         | Equals                                                  |
-| `!=`         | Not equals                                              |
-| `<`          | Less than                                               |
-| `<=`         | Less than or equal to                                   |
-| `>`          | Greater than                                            |
-| `>=`         | Greater than or equal to                                |
-| `contains`   | String, List, or Map contains                           |
-| `startswith` | String or List starts with text or string, respectively |
-| `endswith`   | String or List ends with text or string, respectively   |
-| `match`      | Match string with regular expression                    |
-| `+`          | Add (or string concatenation)                           |
-| `-`          | Subtract                                                |
-| `*`          | Multiply                                                |
-| `/`          | Divide (integer)                                        |
-| `%`          | Modulus                                                 |
+| Operator            | Description                                             |
+| ------------------- | ------------------------------------------------------- |
+| `&&`                | And                                                     |
+| <code> \|\| </code> | Or                                                      |
```

Comment:
This is because `mkdocs` doesn't use [GitHub Flavored Markdown](https://github.github.com/gfm/#tables-extension-) (See mkdocs/mkdocs#2761).

Reply:
Do we care about adding a comment at the top of the table explaining this @pajlada

Reply:
nah